spec/system/page_objects/pages/category.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Pages
+    class Category < PageObjects::Pages::Base
+      # keeping the various category related features combined for now
+
+      def visit(category)
+        Capybara.current_session.visit("/c/#{category.id}")
+        self
+      end
+
+      def visit_settings(category)
+        Capybara.current_session.visit("/c/#{category.slug}/edit/settings")
+        self
+      end
+
+      def back_to_category
+        find('.edit-category-title-bar span', text: 'Back to category').click
+        self
+      end
+
+      def save_category_settings
```

Comment:
Not sure we need category in the name here, or if we do we might want to add it to others